catalogmanagementv1/catalog_management_v1_integration_test.go

```diff
@@ -737,9 +737,9 @@ var _ = Describe(`CatalogManagementV1 Integration Tests`, func() {
 			}
 
 			supportTimeModel := &catalogmanagementv1.SupportTime{
-				Day:       core.Int64Ptr(int64(38)),
-				StartTime: core.StringPtr("testString"),
-				EndTime:   core.StringPtr("testString"),
+				Day:       core.Int64Ptr(int64(30)),
```

Comment:
Just curious how this change is related to the Partner Center Sell service???

Reply:
Sorry i do not understand the question can you ellaborate?